packages/data/addon/services/navi-elide-apollo.ts

```diff
@@ -72,9 +67,7 @@ export default class NaviElideApolloService extends ApolloService {
    */
   _buildURLPath(dataSourceName: string): string {
     const host = configHost({ dataSourceName });
-    const { namespace } = this;
-
-    return `${host}/${namespace}`;
+    return new URL(host, `${document.location.protocol}${document.location.host}`).href;
```

Comment:
Is adding the base intentional? I don't think we expect people to pass `/graphql` as a host

Reply:
In a single server scenario, this is what elide sends:
```js
{
   "user":"testuser",
   "dataSources":[
      {
         "name":"default",
         "uri":"/graphql/api/v1",
         "type":"elide",
         "timeout":900000
      }
   ],
   "appPersistence":{
      "name":"persistence",
      "uri":"/api/v1",
      "type":"elide",
      "timeout":900000
   },
   "FEATURES":{
      "enableDashboardsFilters":true,
      "enableTableEditing":true,
      "enableTotals":true
   }
}
```

Reply:
via https://github.com/yahoo/navi/pull/1110/files#diff-530d4b5a68b71ffccd1308f7347bad17d76dfe298c400b866ab61a68775c63cbR20-R22